From 9f37ec39ef62ab8fb2cc462d9e53bbf435458b5a Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Wed, 19 Apr 2023 16:11:34 +0800
Subject: [PATCH] tvtest: No signal while non-50Hz input [1/1]

PD#SWPL-112677

Problem:
Tvtest does not support screen refresh rates other than 50Hz

Solution:
set panel1_type as vbyone_2

Fixed conflicts:
NA

Verify:
local t7c build

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: Ibeb20d280ca827ccb0d0fdc5f8c78f923f071e3b
---
 board/amlogic/configs/t7_an400_lpddr4x.h        | 2 +-
 board/amlogic/configs/t7_an400_lpddr4x_debian.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/amlogic/configs/t7_an400_lpddr4x.h b/board/amlogic/configs/t7_an400_lpddr4x.h
index bed8d22f8e..3f47f67539 100644
--- a/board/amlogic/configs/t7_an400_lpddr4x.h
+++ b/board/amlogic/configs/t7_an400_lpddr4x.h
@@ -75,7 +75,7 @@
 #else
 #define CONFIG_EXTRA_HDMI_ENV_SETTINGS \
 	"panel_type=vbyone_0\0" \
-	"panel1_type=vbyone_1\0" \
+	"panel1_type=vbyone_2\0" \
 	"panel2_type=lvds_1\0" \
 	"lcd1_ctrl=0x00000000\0" \
 	"lcd2_ctrl=0x00000000\0" \
diff --git a/board/amlogic/configs/t7_an400_lpddr4x_debian.h b/board/amlogic/configs/t7_an400_lpddr4x_debian.h
index 022d959556..af3d89cb38 100644
--- a/board/amlogic/configs/t7_an400_lpddr4x_debian.h
+++ b/board/amlogic/configs/t7_an400_lpddr4x_debian.h
@@ -448,7 +448,7 @@
 		"dv_fw_addr=0xa00000\0"\
 		"otg_device=1\0" \
 		"panel_type=vbyone_0\0" \
-		"panel1_type=vbyone_1\0" \
+		"panel1_type=vbyone_2\0" \
 		"panel2_type=lvds_1\0" \
 		"lcd_ctrl=0x00000000\0" \
 		"lcd1_ctrl=0x00000000\0" \
-- 
2.25.1

